libraries/AC_WPNav/AC_WPNav_Heli.cpp

```diff
@@ -0,0 +1,271 @@
+#include "AC_WPNav_Heli.h"
+#include <AP_HAL/AP_HAL.h>
+
+// table of user settable parameters
+const AP_Param::GroupInfo AC_WPNav_Heli::var_info[] = {
+    // parameters from parent vehicle
+    AP_NESTEDGROUPINFO(AC_WPNav, 0),
+
+    // @Param: USE_L1_NAV
+    // @DisplayName: Waypoint mission use L1 navigation with speed/height control
+    // @Description: This controls if waypoint missions use L1 navigation
+    // @Values: 0:Disable,1:Enable
+    // @User: Standard
+    AP_GROUPINFO("USE_L1_NAV",   1, AC_WPNav_Heli, _l1_nav_use, 0),
```

Comment:
these param names are odd. What happened to "NAVL1_USE" and "NAVL1_LTR_RAD"?

Reply:
@magicrub are you saying that I should put these parameters in my L1 library?  Then I would have to make a call from WPNAV to L1 to determine if the user wants to use L1.  I think that would be ok.